mmv1/templates/terraform/examples/dataplex_datascan_full_profile.tf.erb

```diff
@@ -28,9 +28,26 @@ resource "google_dataplex_datascan" "<%= ctx[:primary_resource_id] %>" {
     exclude_fields {
       field_names = ["property_type"]
     }
+    post_scan_actions {
+      bigquery_export {
+        results_table = "//bigquery.googleapis.com/projects/<%= ctx[:test_env_vars]['project_name'] %>/datasets/dataplex_dataset_%{random_suffix}/tables/profile_export"
```

Comment:
rather than hard-coding `random_suffix` here, please use a variable that includes a hyphen or underscore to automatically generate a resource id for tests that can be automatically cleaned up.

See https://googlecloudplatform.github.io/magic-modules/develop/test/#add-a-create-test for more information.

Reply:
Done. I wasn't sure if i need to use vars v/s test_env_vars in case there's some special handling.

I can change them all to be test_env_vars if it's all the same.

Reply:
`test_env_vars` is specifically for things like project, org, and other things that are provided to the test runner via environment variables :-) `vars` is preferred for things like resource names.